install.sh

```diff
@@ -109,6 +108,7 @@ echo "Install Phosphor CLI environment to ${INSTALL_ROOT}"
 
 THIS_DIR="$(realpath $(dirname $0))"
 
+install -DT --mode 0755 ${THIS_DIR}/clicmd ${INSTALL_ROOT}/usr/bin/clicmd
```

Comment:
The permissions of `${real_file}` are `0400` but of the `clicmd` are `0755`.
Lets increase our paranoia and set `0555`. ;)

Reply:
Let's do it for all executable files in the system! )

Reply:
Yes, make clicmd 555.

Reply:
Done.